# Name of the airplanes 
string[] obstacles_id

# States of the intruders:
gnss_multipath_plugin/AdsbInfo[] intruder_states